test/unit/steps/appointee/appointee-contact-details/Appointee-contact-details.test.js

```diff
@@ -220,8 +220,8 @@ describe('Appointee-contact-details.js', () => {
           expect(field.constructor.name).to.eq('FieldDescriptor');
         });
 
-        it('contains validation', () => {
-          expect(field.validations).to.not.be.empty;
+        it('addressLine2 contains validation', () => {
+          expect(field.validations.length).to.eq(1);
```

Comment:
why here length is 1? its not consistent length check 2

Reply:
No, address line 2 is optional and therefore only has 1 validator